routes/error-tracker.js

```diff
@@ -49,6 +47,39 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
```

Comment:
Minor: Use an active verb for the function name. E.g. `convertStackTrace` or `chromeifyStackTrace`.